content/02.introduction.md

```diff
@@ -1,30 +1,30 @@
 ## Introduction
 
-Pediatric brain and spinal cord tumors are collectively the second most common malignancy in children after leukemia, and they represent the leading disease-related cause of death in children [@doi:10.1093/neuonc/noz150].
+Pediatric brain and spinal cord tumors are collectively the second most common malignancy in children after leukemia, representing the leading disease-related cause of death in children [@doi:10.1093/neuonc/noz150].
 Five-year survival rates vary widely across different histologic and molecular classifications of brain tumors.
 For example, most high-grade gliomas carry a universally fatal prognosis, while children with pilocytic astrocytoma have an estimated 10-year survival rate of 92% [@doi:10.1093/neuonc/now207].
-Moreover, estimates from 2009 suggest that children and adolescents aged 0-19 with brain tumors in the United States have lost an average of 47,631 years of potential life [@doi:10.1002/cam4.410].
+Moreover, estimates from 2009 suggest that children and adolescents aged 0-19 with brain tumors in the United States lose an average of 47,631 years of life [@doi:10.1002/cam4.410].
 
 The low survival rates for some pediatric tumors are clearly multifactorial, explained partly by our lack of comprehensive understanding of the ever-evolving array of brain tumor molecular subtypes, difficulty drugging these tumors, and the shortage of drugs specifically labeled for pediatric malignancies.
```

Comment:
I might stay away from "incomplete characterization" - while this is technically true since one can never be sure when it is complete because we are not all-knowing, it sounds negative toward the field - like they aren't doing a good job, rather than something _we_ are taking ownership for